db/organizations/twitter.eno

```diff
@@ -1,9 +1,9 @@
-name: Twitter
+name: X Corp
```

Comment:
```suggestion
name: X Corp.
```

Looks like the trailing dot is part of their official spelling (source: https://en.wikipedia.org/wiki/X_Corp.)